src/views/List.jsx

```diff
@@ -1,15 +1,53 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	//set state
+	const [searchedItem, setSearchedItem] = useState('');
+	//filtering items searched
+	const filteredItems = data.filter((item) =>
+		item.name.toLowerCase().includes(searchedItem.toLowerCase()),
+	);
+	//setting where to get searched item value
+	function handleChange(e) {
+		setSearchedItem(e.target.value);
+	}
+	//telling the button to clear the search field
+	function buttonClick(e) {
+		setSearchedItem('');
+	}
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			<form>
+				<label htmlFor="filter">Filter Items:</label>
+				<input
+					id="filter"
+					type="text"
+					inputmode="search"
```

Comment:
When I ran this branch on my end, I got an dev console error that said "Warning: Invalid DOM property `inputmode`. Did you mean `inputMode`?"  Might be another instance HTML attributes becoming camelcase in React

Reply:
@louieknolle nice catch, I updated it to `inputMode` and that fixed the error